src/pages/create/set/[id].tsx

```diff
@@ -20,8 +12,24 @@ export default function EditSet() {
   const [setDescription, setSetDescription] = useState("");
   const [flashcards, setFlashcards] = useState<TermDefPair[]>([]);
   const setId = useRouter().query.id as string;
+  const isEdit = !!(useRouter().query?.isEdit as string);
 
-  const { data: cards } = api.card.geCardBySet.useQuery(
+  const updateCard = ({
+    id,
+    term,
+    def,
+  }: {
+    id: number;
+    term: string;
+    def: string;
+  }) => {
+    if (flashcards?.[id]) {
+      flashcards[id].term = term;
+      flashcards[id].def = def;
+    }
+  };
+
+  const { data: cards } = api.card.getCardsBySet.useQuery(
```

Comment:
I feel like you should be able to get the cards from api.set.getOneSet.useQuery because that query includes that card objects so hopefully should be easier and faster to just use one query